components/feature/toolbar/src/main/java/mozilla/components/feature/toolbar/WebExtensionToolbarFeature.kt

```diff
@@ -87,20 +87,8 @@ class WebExtensionToolbarFeature(
                     toolbarAction
                 }
 
-                // Check if we have a tab-specific override and apply it.
-                // Note that tab-specific actions only contain the values that
-                // should be overridden. All other values will be null and should
-                // therefore *not* be applied / overridden.
```

Comment:
nit: Let's add a comment here `// Apply tab-specific override of browser action`